ortools/routing/samples/cvrp_reload.py

```diff
@@ -334,31 +334,34 @@ def print_solution(
     for vehicle_id in range(data["num_vehicles"]):
         index = routing.Start(vehicle_id)
         plan_output = f"Route for vehicle {vehicle_id}:\n"
+        load_value = 0
         distance = 0
         while not routing.IsEnd(index):
-            load_var = capacity_dimension.CumulVar(index)
             time_var = time_dimension.CumulVar(index)
             plan_output += (
                 f" {manager.IndexToNode(index)} "
-                f"Load({assignment.Min(load_var)}) "
+                f"Load({assignment.Min(capacity_dimension.CumulVar(index))}) "
                 f"Time({assignment.Min(time_var)},{assignment.Max(time_var)}) ->"
             )
             previous_index = index
             index = assignment.Value(routing.NextVar(index))
             distance += distance_dimension.GetTransitValue(previous_index, index, vehicle_id)
-        load_var = capacity_dimension.CumulVar(index)
+            # capacity dimension TransitVar is negative at reload stations during replenishment
+            # don't want to consider those values when calculating the total load of the route
+            # hence only considering the positive values
+            load_value += max(0,capacity_dimension.GetTransitValue(previous_index,index,vehicle_id))
```

Comment:
Why += and not =

Reply:
inside the `while not routing.IsEnd(index):` accumulating non negative (aka not reloading) transitValue...

Reply:
OK